docs/content/querying/sql.md

```diff
@@ -571,6 +571,8 @@ The "sys" schema provides visibility into Druid segments, servers and tasks.
 ### SEGMENTS table
 Segments table provides details on all Druid segments, whether they are published yet or not.
 
+#### CAVEAT
+Note that a segment can be served by more than one realtime or historical servers, in that case it would have multiple replicas. These replicas are weakly consistent with each other when served by multiple realtime tasks, until a segment is eventually served by a historical, at that point the segment is immutable. And broker prefers to query a segment from historical over realtime server. But if a segment has multiple realtime replicas, for eg. kafka index tasks, and one task is slower than other, then the sys.segments query results can vary for the duration of the tasks. The columns of segments table that can have inconsistent values during this period include `size`, `num_replicas`, `num_rows`.
```

Comment:
Would you explain why `size` and `num_replica` vary? It looks that they are not getting from segmentMetadataQuery.

Reply:
Please add why this happens. The root cause is that system schema uses segmentMetadatQuery to retrieve some information, and the broker randomly picks one of the realtime tasks for query processing if there's no published segments, and thus it's not guaranteed that the same task serves segmentMetadataQuery every time.

I think it's worth to link https://github.com/apache/incubator-druid/issues/5915 here too.

Reply:
hmm, should there be mention of `segmentMetadatQuery` and `RandomServerSelectorStrategy` in user facing docs. I tried to explain without adding internal code details. I feel such details should be in github issues or in javadocs. And do we generally link to github issues in user documentation, are there any similar examples in druid docs?

Reply:
SegmentMetdataQuery is a documented query type (http://druid.io/docs/latest/querying/segmentmetadataquery.html). I don't think it's worth to mention the class name of `RandomServerSelectorStrategy` but the configuration for it is also documented (http://druid.io/docs/latest/configuration/index.html#query-prioritization). 

Well, but my above comment about random selection may not be appropriate because it can give a wrong intuition to users. Probably better to not say about random selection at all. But, I think it's still needed to say about only one of the realtime tasks is selected if multiple replicas are running.

Reply:
> And do we generally link to github issues in user documentation, are there any similar examples in druid docs?

Why not? Here're some examples: https://cse.google.com/cse?cx=000162378814775985090%3Amolvbm0vggm&q=github&oq=github&gs_l=partner-generic.3...1401.2048.0.2184.6.6.0.0.0.0.102.536.5j1.6.0.gsnos%2Cn%3D13...0.606j90652j6...1.34.partner-generic..5.1.102.mApbmyfw_Jw.

Reply:
Hmm. For `num_replica`, it sounds like it's a valid result because it reflects the changes which actually happened. I think it's different from varying `num_rows` and doesn't have to be noted here.

Reply:
In that case,  it seems `num_rows` is the only col affected.